manifests/v/Victoria/Victoria/5.37/Victoria.Victoria.installer.yaml

```diff
@@ -1,20 +1,20 @@
-# Created with komac v2.2.1
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
-
-PackageIdentifier: Victora.HDD-SSD
-PackageVersion: '5.37'
-InstallerType: zip
-NestedInstallerType: portable
-NestedInstallerFiles:
-- RelativeFilePath: Victoria537/Victoria.exe
-InstallModes:
-- silent
-- silentWithProgress
-UpgradeBehavior: install
-ReleaseDate: 2021-10-14
-Installers:
-- Architecture: x86
-  InstallerUrl: https://hdd.by/?download=1&kccpid=916&kcccount=https://hdd.by/Victoria/Victoria537.zip
-  InstallerSha256: A66B7869D59BBF90B5994332BAC57BCBC0B377CEE9F0C59C078795EA7B4C99D1
-ManifestType: installer
-ManifestVersion: 1.6.0
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Victoria.Victoria
```

Comment:
Automatic Validation ended with: 

>  Addition and deletion only permitted in the same package directory.

The pipelines identify manifest by `PackageIdentifier`, so this looks like a deletion of `Victoria.Victoria` and an addition of `Victora.HDD-SSD` in the same PR. This would disagree with the limit of 1 manifest (add/modify/remove) within the same PR. 

Could one of these changes be moved to another PR? Then both PRs should be able to pass this check.

Reply:
Ah I see, let me fix this then